.github/workflows/validate-test.yml

```
@@ -0,0 +1,36 @@
+
+            -   name: Set up JDK 17
+                uses: actions/setup-java@v3
+                with:
+                    java-version: 17
+                    distribution: 'temurin'
+
+            -   name: Install docker-compose
+                run: |
+                    apt-get update && apt-get install -y docker-compose
+
+            -   name: Prepare Test Environment
```

YAMLlint (1.35.1)

[warning] too many spaces after hyphen

(hyphens)